docs/before/accounts.md

```diff
@@ -0,0 +1,101 @@
+
+# Step 2 - Access and process data
+Now that you can access the computer systems at SLAC, it is time to get yourself oriented. 
+It is useful for that to understand how the data flows from the beamline to storage, *a.k.a* to understand the LCLS Data Systems, built with 3 successive levels:
+
+- **1 &#x3e;  DAQ+DRP - Data acquisition and reduction**:<br>
+provides diagnostics within <1s. *(Experts only)*
+- **2 &#x3e;  FFB - Fast Feedback**:<br> 
+provides diagnostics and results within <1min. *(All)*
+- **3 &#x3e;  Offline Analysis**:<br> 
+for intensive compute during and after the beamtime. *(All)*
+
+*TODO: figure to describe the network and data infrastructure.*
+
+!!! danger "FFB migration to S3DF"
+    S3DF is a new capability at SLAC and is currently still a work in progress. 
+    The FFB nodes will soon be a part of it - when that happens, it will be more seamless to switch from online analysis to offline analysis at the end of the beamtime. 
+    But for now (March 2023), the approach described in the first section below is still valid for past experiments.
+    In time, experiments will be analyzed on S3DF both during and after beamtimes (see following section).
+
+### &#x2192; on the FFB - Fast Feedback System {: #ffb}
+
+First, get in the system:
+```
+ssh <username>@pslogin.slac.stanford.edu
+```
+From there, you can access the FFB nodes:
+```
+ssh psffb
+```
+The experiments and their data can be found at the following path:
+```
+/cds/data/drpsrcf/<instrument>/<experiment>
+```
+!!! danger "Data import/export"
+    Note that the FFB nodes are not exposed to external networks. If you need to transfer data, you can go through `psexport.slac.stanford.edu`.
+
+*TODO: a few sentences on batch, interactive and jupyter.*
+
+
+!!! note "More FFB info"
+    For more information about the Fast Feedback System, check out the PCDS documentation [here][8].
+
+
+### &#x2192; at S3DF - SLAC Shared Science Data Facility {: #s3df}
+
+First, get in the system:
+```
+ssh <username>@s3dflogin.slac.stanford.edu
+```
+In order to get access to the data and to submit SLURM jobs, reach an interactive node:
+```
+ssh psana
+```
+Once on the interactive node, execute the following command (or place it in your `~/.bashrc`) in order to set the necessary environment variables and environments that will allow you to find the data and process it:
+``` 
+source /sdf/group/lcls/ds/ana/sw/conda1/manage/bin/psconda.sh
+```
+!!! note "FFB at S3DF"
+    In S3DF, both FFB and Offline storage are accessible. By default, the command above points to the offline storage. In order to access the FFB storage, you can override the environment variable that points to the data:
+    ```
+    export SIT_PSDM_DATA=/sdf/data/lcls/drpsrcf/ffb
+    ```
+The experiments and their data can be found at the following path:
+```
+${SIT_PSDM_DATA}/<instrument>/<experiment>
+``` 
+
+*TODO: a few sentences on batch, interactive and jupyter.*
```

Comment:
Reading it now, I think that it will probably be worth having a separate page on SLURM jobs/batch system?

Maybe not, but I feel it may be easier to digest:
- This page = getting an account, understanding nodes, accessing data
- batch page = submitting intensive jobs, basic commands (squeue, scancel etc)

Reply:
Good idea.

Reply:
Created Issue #15